index.html

```diff
@@ -577,7 +571,7 @@ <h4 id="bidiCase1">Final punctuation</h4>
   <p lang="he" dir="rtl" style="font-size: 1.8em; color: grey;">תוצאה: "בינלאומי!"</p>
 
 <p>The Hebrew characters are automatically displayed right-to-left by applying the Unicode Bidirectional Algorithm (UBA). However, in a LTR context the UBA cannot make the exclamation mark appear to the left of the Hebrew text, where it belongs, unless the base direction is set to RTL around the inserted string.</p>
-<p>In HTML this can be done by inserting the string into a <code class="kw" translate="no">dir</code> attribute with the value <code class="kw" translate="no">rtl</code>. That yields the following:</p>
+<p>In HTML this can be be achieved for a LTR context by inserting the string into a <code class="kw" translate="no">dir</code> attribute with the value <code class="kw" translate="no">rtl</code>. That yields the following:</p>
```

Comment:
Two thoughts here:

1. "...inserting the string into a `dir` attribute with the value..." reads oddly. Probably it should say "...inserting the string into an element with a `dir` attribute with the value..."

2. I'm not sure your edit adds anything? It's true that in an RTL context this is kind of a no-op---provided there aren't other bidi issues involved. In HTML, the dir attribute also activates bidi isolation, so a case like an LTR run at the end (and outside) the quoted text would also be solved. Consider

> ברוך הבא ל- W3C!

Reply:
Edit 1 made. Thanks for catching.

Wrt point 2, i wanted to indicate the context of the example. As you say, the implications are different for RTL contexts, and i didn't want to get into that complexity but i did want to ensure that my frame of reference is clearly delineated.